lib/sparql.jison

```diff
@@ -628,7 +669,25 @@ TriplesTemplate
     ;
 GroupGraphPattern
     : '{' SubSelect '}' -> { type: 'group', patterns: [ $2 ] }
-    | '{' GroupGraphPatternSub '}' -> { type: 'group', patterns: $2 }
+    | '{' GroupGraphPatternSub '}' 
+    { 
+      // For every binding
+      for (let binding of $2.filter(el => el.type === "bind")) {
+        const index = $2.indexOf(binding);
+        let bindedVars = [];
```

Comment:
```suggestion
        const boundVars = [];
```

but we probably want to make it a `Set` given that is used for membership checking